crates/http-sink/src/sink.rs

```diff
@@ -14,7 +16,10 @@ pub(crate) struct HttpSink {
 
 impl HttpSink {
     pub(crate) fn new(config: &HttpConfig) -> Result<Self> {
-        let client = Client::new();
+        let client = Client::builder()
+            .timeout(Duration::from_secs(15))
+            .connect_timeout(Duration::from_secs(5))
```

Comment:
probably we should add this in HttpConfig

Reply:
We can use humantime_serde. And store directly as Duration on config

See:

https://github.com/infinyon/http-source-connector/blob/bc76bddbe01207c1723c6a0a3ad108c6466ad9b4/crates/http-source/src/config.rs#L30

Reply:
my bad actually I got it to work. I was confused by the error

Reply:
I think that in order to make it similar to http-source. we can make it accept a human readable string. with `humantime_serde`. I think that way it is more easy to use for users. Remember that this configuration at the end is only used by end users when they are writing the connector configuration yaml

Reply:
done